### ISSUE TITLE
Enhance News Board

### ISSUE THREAD

**Issue by pkrshnan** [uwhvz/uwhvz#7]
The news board currently goes largely unused. It should be much easier to use, such that any necro can easily pin mission details there.

**Comment by firemaster1294** [comment]
Wagtail is a bit of a nightmare to learn to use as Necro. Once you get the hang of it, it works *meh*, but this needs to be much better to use

**Comment by pkrshnan** [comment]
Yeah I'll take a look on what exactly it entails to improve Wagtail. As a first step, I'll likely try to document the process in publishing posts, and then make the overall process _way_ easier. It's far too confusing to use right now.
